developer_setup.md

```diff
@@ -211,6 +211,22 @@ bin/setup
 
 macOS requires platform specific Gems. Run `bundle config specific_platform true` before running `bin/setup`.
 
+The ovirt-engine-sdk gem contains C code that generated warnings with older CLANG versions.  These warnings are marked as errors in newer CLANG 15 and 16 depending on the CLANG version, architecture, and possibly OS.  If this is the case, you can mark these as warnings and not errors to allow installation:
+
+```
+gem install ovirt-engine-sdk -v4.6.0 -- --with-cflags="-Wno-error=incompatible-function-pointer-types -Wno-error=implicit-function-declaration"
+```
+
+This can also be configured within bundler:
+
+```
+bundle config build.ovirt-engine-sdk --with-cflags="-Wno-error=incompatible-function-pointer-types -Wno-error=implicit-function-declaration"
+```
+
+See https://github.com/oVirt/ovirt-engine-sdk-ruby/issues/14 and https://github.com/oVirt/ovirt-engine-sdk-ruby/issues/11
+
+Hopefully, the code in the gem can be resolved to avoid these warnings in the future.
+
 If you've run PostgreSQL in a container, be sure to export the `DATABASE_URL` variable to connect to the container over TCP instead of a UNIX file socket.
 
 ```bash
```

Comment:
This section is weird cause now it has 3 separate notes that run into each other, so it's hard to differentiate the individual problems...I'm thinking this should be a bullet (and the other 2 be bullets as well, but the GitHub UI won't let me do that in this suggestion - but I think you get the point).

```suggestion
- macOS requires platform specific Gems. Run `bundle config specific_platform true` before running `bin/setup`.

- The ovirt-engine-sdk gem contains C code that generated warnings with older CLANG versions.  These warnings are marked as errors in newer CLANG 15 and 16 depending on the CLANG version, architecture, and possibly OS.  If this is the case, you can mark these as warnings and not errors to allow installation:

  ```
  gem install ovirt-engine-sdk -v4.6.0 -- --with-cflags="-Wno-error=incompatible-function-pointer-types -Wno-error=implicit-function-declaration"
  ```

  This can also be configured within bundler:

  ```
  bundle config build.ovirt-engine-sdk --with-cflags="-Wno-error=incompatible-function-pointer-types -Wno-error=implicit-function-declaration"
  ```

  See https://github.com/oVirt/ovirt-engine-sdk-ruby/issues/14 and https://github.com/oVirt/ovirt-engine-sdk-ruby/issues/11

  Hopefully, the code in the gem can be resolved to avoid these warnings in the future.

- If you've run PostgreSQL in a container, be sure to export the `DATABASE_URL` variable to connect to the container over TCP instead of a UNIX file socket.

  ```bash
```